packages/chord-mark/tests/unit/renderer/components/renderSong.spec.js

```diff
@@ -690,3 +690,43 @@ describe('Custom ChordSymbol renderer', () => {
 		expect(toText(rendered)).toBe('|Custom     |Custom     |Custom     |');
 	});
 });
+
+describe('Wrap Sections in Divs', () => {
+	test('Correctly create and class sections', () => {
```

Comment:
Can you add a test where no section is declared at all to make sure that no extra markup is created?